styles/style.css

```diff
@@ -0,0 +1,412 @@
+    background-repeat: no-repeat;
+    background-position: center;
+}
+
+.carousel__slide:nth-child(1):before {
+    content: "as scientist, 2017";
+    position:absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate3d(-50%,-40%,70px);
+    color: rgba(255, 255, 255, 0.638);
+    font-size: 5em;
+}
+.carousel__slide:nth-child(2):before {
+    content: "as sales specialist, 2018";
+    position:absolute;
+    top: 50%;
+    left: 50%;
+    transform: translate3d(-50%,-40%,70px);
+    color: rgba(255, 255, 255, 0.83);
+    font-size: 5em;
+}
+.carousel__slide:nth-child(3):before {
+    content: "as technical advisor, 2020";
+    position:absolute;
+    top: 60%;
+    left: 45%;
+    transform: translate3d(-50%,-40%,70px);
+    color: rgba(255, 255, 255, 0.956);
+    font-size: 5em;
+}
+.carousel__slide:nth-child(4):before {
+    content: "as software developer, 2022";
+    position:absolute;
+    top: 70%;
+    left: 50%;
+    transform: translate3d(-50%,-40%,70px);
+    color: rgba(255, 255, 255, 0.638);
+    font-size: 5em;
+}
+.carousel__snapper {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    scroll-snap-align: center;
+}
+
+@media (hover: hover) {
+    .carousel__snapper {
+    animation-name: tonext, snap;
+    animation-timing-function: ease;
+    animation-duration: 4s;
+    animation-iteration-count: infinite;
+    }
+
+.carousel__slide:last-child .carousel__snapper {
+    animation-name: tostart, snap;
+    }
+}
+
+@media (prefers-reduced-motion: reduce) {
+    .carousel__snapper {
+    animation-name: none;
+    }
+}
+
+.carousel:hover .carousel__snapper,
+.carousel:focus-within .carousel__snapper {
+    animation-name: none;
+}
+
+.carousel__navigation {
+    position: absolute;
+    right: 0;
+    bottom: 0;
+    left: 0;
+    text-align: center;
+}
+
+.carousel__navigation-list,
+.carousel__navigation-item {
+    display: inline-block;
+}
+
+.carousel__navigation-button {
+    display: inline-block;
+    width: 1.5rem;
+    height: 1.5rem;
+    background-color: #333;
+    background-clip: content-box;
+    border: 0.25rem solid transparent;
+    border-radius: 50%;
+    font-size: 0;
+    transition: transform 0.1s;
+}
+
+.carousel::before,
+.carousel::after,
+.carousel__prev,
+.carousel__next {
+    position: absolute;
+    top: 0;
+    margin-top: 37.5%;
+    width: 4rem;
+    height: 4rem;
+    transform: translateY(-50%);
+    border-radius: 50%;
+    font-size: 0;
+    outline: 0;
+}
+
+.carousel::before,
+.carousel__prev {
+    left: -1rem;
+}
+
+.carousel::after,
+.carousel__next {
+    right: -1rem;
+}
+
+.carousel::before,
+.carousel::after {
+    content: '';
+    z-index: 1;
+    background-color: #333;
+    background-size: 1.5rem 1.5rem;
+    background-repeat: no-repeat;
+    background-position: center center;
+    color: #fff;
+    font-size: 2.5rem;
+    line-height: 4rem;
+    text-align: center;
+    pointer-events: none;
+}
+
+.carousel::before {
```

Comment:
Heads up that using `::before` to add content is [discouraged due to accessibility concerns](https://developer.mozilla.org/en-US/docs/Web/CSS/::before#accessibility_concerns).